nostr/event.py

```diff
@@ -23,7 +23,7 @@ def __init__(
             content: str, 
             created_at: int = None, 
             kind: int=EventKind.TEXT_NOTE, 
-            tags: "list[list[str]]"=[], 
```

Comment:
this is better done with default_factory see: https://github.com/jeffthibault/python-nostr/pull/84